### PR TITLE
Sus 751

### PR DESCRIPTION
**Pull request by justnpT**
Please don't merge this branch until SUS-194 is resolved fixed

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/oasis/components/comment/ArticleComment.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 import com.gargoylesoftware.htmlunit.ElementNotFoundException;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/oasis/components/comment/ArticleComment.java]: MINOR Remove this unused import 'com.gargoylesoftware.htmlunit.ElementNotFoundException'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/elements/oasis/components/comment/ArticleComment.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
...
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/oasis/components/comment/ArticleComment.java]: MINOR Remove this unused import 'org.openqa.selenium.WebDriver'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by ludwikkazmierczak** [comment]
LGTM, before merge, please fix unused imports

**Pull request merged**